src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -946,8 +1300,8 @@ const Authenticate = (() => {
             account_status = response.get_account_status;
             const { needs_verification } = account_status.authentication;
             
-            const type_required = type === 'identity' ? 'poi' : 'poa';
-            const type_pending = type === 'identity' ? 'poa' : 'poi';
+            const type_required = type === 'identity' ? 'poi' : type === 'income' ? 'edd' : 'poa';
```

Comment:
May be its better to use switch-case or if-else here so it can be expanded in future?

Reply:
agree with you. look a little bit overcomplicated

Reply:
@likhith-deriv @yauheni-kryzhyk-deriv after checking the codes i'll remove the `edd` option since we are not passing income types to the `showCTAButton` function. Thanks for the suggestion 🙏